code/script.js

```diff
@@ -225,101 +232,164 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log('secret person selected')
+  console.log(secret)
```

Comment:
With `console.log(secret)` anyone who goes into Inspector can cheat ;) It's a good tool to have while testing your filtering, but then it should be removed.

Reply:
Thanks, completely forgot and will remove.